app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,21 @@
+class AnswersController < ApplicationController
+
+  def create
+    @question = Question.find(params[:question_id])
+    @answer = @question.answers.build(answer_params)
+
+    if @answer.save
+      flash[:notice] = 'Answer was added'
+    else
+      flash[:notice] = 'Something goes wrong'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.